packages/portal/src/pages/search/index.vue

```diff
@@ -5,16 +5,38 @@
     >
       <template
         v-if="!!searchQuery"
-        #related
+        #related-galleries
       >
-        <client-only>
-          <RelatedSection
-            :query="searchQuery"
-            :overrides="relatedCollections"
-            data-qa="related section"
-            @fetched="handleRelatedSectionFetched"
-          />
-        </client-only>
+        <aside
```

Comment:
Can we instead make `aside` the root element inside the RelatedSection and RelatedGalleries components?

That should prevent empty aside elements being rendered when there is nothing related, e.g. no related galleries to "wolf".

Reply:
I have moved the `aside` to the RelatedSection and RelatedGalleries components but without adding the conditional render. We either need to
- keep the aside placeholder for the related cards to reserve the space in the masonry, also when there are no related results, otherwise it will still cause a jump in the layout to fill the empty space
- Wait for everything to be ready and then show the masonry layout. We'd need to do state management to check if all components in the masonry are ready. To me this seems a bit overtly complex, harder to maintain, longer time before anything is visible and error prone.
- accept a jumpy layout, maybe tweaking the transition a bit to smudge the asynchronous loading times.